pkg/openstack/loadbalancer.go

```diff
@@ -605,6 +606,32 @@ func (lbaas *LbaasV2) getLoadBalancerLegacyName(_ context.Context, _ string, ser
 	return cloudprovider.DefaultLoadBalancerName(service)
 }
 
+// this function gets the listeners tags from the service annotation and   
+func (lbaas *LbaasV2) customLoadBalanceListenerTag(service *corev1.Service) ([]string, error) {
+	var listener listeners.Listener
+	annotationVal := getStringFromServiceAnnotation(service, ServiceAnnotationLoadBalancerListenerTags, "")
+
+	listOfTags := strings.Split(annotationVal, "=")
+
+	if annotationVal == "" {
+		return listener.Tags, fmt.Errorf("annotation is missing or empty for %q", service.Name)
+	}
+
+	if annotationVal != "" {
```

Comment:
this condition doesn't make sense, please remove it.

Reply:
Why is this marked as resolved?

Reply:
because i fixed it in the code 
is there an issue about the resolving of the comment after fix ??

Reply:
I don't see this fixed in the code, you would want to recheck if you even pushed the commit or now https://github.com/kubernetes/cloud-provider-openstack/blob/97a664a539ff6cfc9414068da04464f540c9c7a0/pkg/openstack/loadbalancer.go#L620

Reply:
you should see it's a WIP and i haven't made recent push of the changes i made
thank you helping 
i appreciate you